### ISSUE TITLE
GDAL import vs load from template puts lines in different place

### ISSUE THREAD

**Issue by mitchellkrome** [OpenOrienteering/mapper#811]
When I load data to a template then import that template to the main map, lines appear as they should.
When I use the import button to directly load the the file into the map, the lines appear in some random place all on top of each other.

The only difference I can see is the template based imported copies the georeferencing from the main map into the temporary map it imports the file into. Running with this patch
```diff
diff --git a/src/map_editor.cpp b/src/map_editor.cpp
index eb16bc5..a370474 100644
--- a/src/map_editor.cpp
+++ b/src/map_editor.cpp
@@ -3843,6 +3843,7 @@ bool MapEditorController::importMapFile(const QString& filename, bool show_error
 {
        Map imported_map;
        imported_map.setScaleDenominator(map->getScaleDenominator()); // for non-scaled geodata
+       imported_map.setGeoreferencing(map->getGeoreferencing());
        
        bool result = imported_map.loadFrom(filename, window, nullptr, false, show_errors);
        if (result)
```

makes things work, but there is code in map::importMap (which is called just after this diff) which I think should have dealt with this case of the georeferencing not matching (it has a todo to test it on however).

I don't know if this is a fix for the issue or just a hack for something else that is broken. 

### Configuration
Mapper Version: master/HEAD with GDAL enabled
Operating System: Ubunto 14.04

**Comment by dg0yt** [comment]
I would prefer to have test files with clear steps.

**Comment by mitchellkrome** [comment]
Empty map with georeferencing is set up as

![georef](https://cloud.githubusercontent.com/assets/8371236/21079277/36280ec6-bfd9-11e6-9675-b4e0ef0703a4.PNG)

OSM file used is [map(1).zip](https://github.com/OpenOrienteering/mapper/files/644351/map.1.zip)

After importing as template I get this:

![template](https://cloud.githubusercontent.com/assets/8371236/21079285/7eb11ff2-bfd9-11e6-89e8-0b6cc89e70d4.PNG)

After importing the same file using import button from "file" menu I get (note the coordinates down the bottom have gone crazy and everything is just in one spot)

![afterimport](https://cloud.githubusercontent.com/assets/8371236/21079286/884600fa-bfd9-11e6-89fe-2cecb6cdef25.PNG)

If I use the existing template and use the "import and remove" button I get 

![afterimportfromtemplate](https://cloud.githubusercontent.com/assets/8371236/21079292/ac95eb50-bfd9-11e6-9845-849febd38f82.PNG)

**Comment by dg0yt** [comment]
I can confirm the issue in HEAD, no matter if using the new (GDAL-based) or old importer. The stable release is not affected.

**Comment by mitchellkrome** [comment]
Did you test old importer by recompiling without GDAL? Because the import button still uses GDAL importer even if you turn it off in settings from my experience looking into this issue.

**Comment by dg0yt** [comment]
Indeed, changing the settings does not have an effect, but HEAD builds without GDAL are not affected.

**Comment by dg0yt** [comment]
91c1868 fixes the GDAL settings issue. Seems like nobody really tries the unstable builds.

The original issue is not that easy to solve, but the major missing piece for (GDAL) OGR vector data import in Mapper: We must look at the georeferencing information of the imported data and determine how to transfer this to our data. Probably sort-of what `ogr2ogr?` does. It may be easy for OSM and GPX, but not in the general case. All my testing was with a non-georeferenced DXF file.

**Comment by dg0yt** [comment]
This issue seems to have been resolved in v0.7.0.
